CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Features
 
+* [\#10963](https://github.com/cosmos/cosmos-sdk/pull/10963) Add `MsgCreateGroupWithPolicy` to group module.
```

Comment:
I don't think we actually need to add this
The very first release with the group module will already have `MsgCreateGroupWithPolicy` so I wouldn't add it to the changelog
Any guidelines on this? @amaurym @robert-zaremba